src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+    this.hidden = hidden;
```

Comment:
The constructor of the Herbivore class should not accept 'health' as a parameter since it defaults to 100. Also, 'hidden' is not defined before being used. You should initialize 'hidden' to 'false' within the constructor.